### PR TITLE
Initial implementation of camera contrast control

### PR DESCRIPTION
**Pull request by KeenS**
## Check list

- [ ] I wrote [CHANGELOG.md](./CHANGELOG.md) if the pull request adds/modifies features.

## Summary
This patch provides initial implementation of contrast (gamma) control. This contains naive implementation of gamma and not contains `ce_enable` feature and reading config from files feature.

### CODE REVIEW

File: actfw_core/unicam_isp_capture.py [hunk 1]
```diff
@@ -75,6 +85,15 @@ def __init__(
...
         self.target_Y: float = target_Y
+        # - update by contrast
+        self.brightness: float = brightness
+        self.contrast: float = contrast or 1.0
+        # self.lo_histogram: float = 0.01
+        # self.lo_level: float = 0.015
+        # self.lo_max: int = 500
+        # self.hi_histogram: float = 0.95
+        # self.hi_level: float = 0.95
+        # self.hi_max: int = 2000
```
Comment (KeenS) [inline actfw_core/unicam_isp_capture.py:96]: following patch will use these variables

File: actfw_core/unicam_isp_capture.py [hunk 2]
```diff
@@ -42,6 +49,8 @@ def __init__(
         init_controls: List[str] = _EMPTY_LIST,
         agc: bool = True,
         target_Y: float = 0.16,  # Temporary set for the developement of agc algorithm
+        brightness: float = 0.0,
+        contrast: Optional[float] = 1.0,
```
Comment (ishiy1993) [inline actfw_core/unicam_isp_capture.py:53]: I think it's better to define `DEFAULT_CONTRAST = 1.0` and use it.

File: actfw_core/unicam_isp_capture.py [hunk 3]
```diff
@@ -75,6 +85,15 @@ def __init__(
         self.degital_gain: float = 1.0  # Currently, this value is constant.
         self.agc_interval_count: int = 0
         self.target_Y: float = target_Y
+        # - update by contrast
+        self.brightness: float = brightness
+        self.contrast: float = contrast or 1.0
```
Comment (ishiy1993) [inline actfw_core/unicam_isp_capture.py:90]: use `DEFAULT_CONTRAST`.

File: actfw_core/unicam_isp_capture.py [hunk 4]
```diff
@@ -58,6 +67,7 @@ def __init__(
         )
         self.do_awb = auto_whitebalance
         self.do_agc = agc
+        self.do_contrast = not not contrast
```
Comment (ishiy1993) [inline actfw_core/unicam_isp_capture.py:70]:
```suggestion
        self.do_contrast = contrast is not None
```

**Review by ishiy1993 — CHANGES_REQUESTED** [review]

**Review by ishiy1993 — APPROVED** [review]
LGTM

**Pull request merged**